### DISCUSSION TITLE
Adding --restic-timeout flag to deployment produces error

### DISCUSSION THREAD

**Discussion by MrGeorgo404** [vmware-tanzu/velero#6777] [Community support Q&A] [answered]
Hello, 

my goal is to edit restic timeout setting which should be by default 240m based on documentation. I want to change it to 60m and therefore I followed the documentation here:  https://velero.io/docs/v1.4/customize-installation/#update-resource-requests-and-limits-after-install .

There is mentioned that I should use --restic-timeout flag on an already existing deployment which changes the default configuration. However, when I add this flag to my deployment Velero pod gets stuck in CrashBackOffLoop: 
![Screenshot 2023-09-06 at 11 40 21](https://github.com/vmware-tanzu/velero/assets/81189094/f02daa1e-5a49-4c13-becd-bce9538715d3)

When I try to look at the logs of the pod I get this: 
![Screenshot 2023-09-06 at 11 39 52](https://github.com/vmware-tanzu/velero/assets/81189094/c069c188-1fbe-4536-b4ca-ee90035a98ae)

It looks like the --restic-timeout flag is not recognized. When I remove this flag from the deployment everything starts to work correctly once again. How should I proceed to change the restic timeout setting then?

**Comment by sseago** [comment, marked as answer]
1) restic-timeout is no longer valid -- the doc you referenced was from Velero 1.4. The current name for the field is fs-backup-timeout: https://velero.io/docs/v1.11/customize-installation/#update-resource-requests-and-limits-after-install
2) it's probably better to put the flag *after* the server line, although it may not matter
